packages/node_modules/@webex/internal-plugin-devices/src/interceptors/device-url.js

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {Interceptor} from '@webex/http-core';
+import {set} from 'lodash';
+
+/**
+  * Adds 'cisco-device-url' header, as appropriate, to requests
+  */
+export default class DeviceUrlInterceptor extends Interceptor {
+  /**
+   * @returns {DeviceUrlInterceptor}
+   */
+  static create() {
+    /* eslint no-invalid-this: [0] */
+    return new DeviceUrlInterceptor({webex: this});
+  }
+
+  /**
+   * @see Interceptor#onRequest
+   * @param {Object} options
+   * @returns {Object}
+   */
+  onRequest(options) {
+    const {headers, service, uri} = options;
+    const {device, services} = this.webex.internal;
```

Comment:
This should be `devices` for this plugin since the namespace of the plugin is `devices`.